packages/account/src/utils/Constants.ts

```diff
@@ -1,8 +1,8 @@
 import { ethers } from 'ethers'
 import { ChainId } from '@biconomy/core-types'
 
-export const DEFAULT_VERIFICATION_GAS_LIMIT = ethers.BigNumber.from(70000);
-export const DEFAULT_CALL_GAS_LIMIT = ethers.BigNumber.from(35000);
-export const DEFAULT_PRE_VERIFICATION_GAS = ethers.BigNumber.from(21000);
+export const DEFAULT_VERIFICATION_GAS_LIMIT = ethers.BigNumber.from(300000)
```

Comment:
These numbers should be reviewed if they make sense for each network and if they matches the real transactions.